### DISCUSSION TITLE
Randomly Not Opening.

### DISCUSSION THREAD

**Discussion by NirZhor3821** [SteamClientHomebrew/Millennium#145] [General]
Hello There,
Im Using Millennium For Steam in Windows 10 Latest. Today I Found Out Millennium is Not Working When I Turn Off My PC. I Have To Install It Again Use It.

**Comment by ejalxndr** [comment]
Even after restarting the Steam client?

**Reply by NirZhor3821** [reply]
Lets Say I Turned Off My PC. After Some Times I Turned It On. When I Start Steam Millennium is Not There. Its Not Working. But When I Do Re-Install It Works Again.

**Reply by ejalxndr** [reply]
> Lets Say I Turned Off My PC. After Some Times I Turned It On. When I Start Steam Millennium is Not There. Its Not Working. But When I Do Re-Install It Works Again.

Whenever the issue presents itself, upload `"C:\Program Files (x86)\Steam\logs\webhelper.txt"` and `"C:\Program Files (x86)\Steam\ext\data\logs\debug.log"`

It should help me locate the error!

**Reply by NirZhor3821** [reply]
Hey It Happened Again. I Closed The Steam Then Reopen It. It Works Fine.

**Reply by DKTGM** [reply]
I have a whole other issue, for some reason the Millennium site doesn't want to work for me.
![Screenshot (2)](https://github.com/user-attachments/assets/6a8c35a7-52f3-4a99-b533-f464daf15232)

**Reply by ejalxndr** [reply]
> I have a whole other issue, for some reason the Millennium site doesn't want to work for me. ![Screenshot (2)](https://private-user-images.githubusercontent.com/134449407/413634382-6a8c35a7-52f3-4a99-b533-f464daf15232.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzk3NDI3MzUsIm5iZiI6MTczOTc0MjQzNSwicGF0aCI6Ii8xMzQ0NDk0MDcvNDEzNjM0MzgyLTZhOGMzNWE3LTUyZjMtNGE5OS1iNTMzLWY0NjRkYWYxNTIzMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDIxNlQyMTQ3MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OWQ3YTNhODQ2OTc4ZGE0ZWY0ZDBhZjcxODYwNmE4NjA3ZjIyMDM1N2YyMTY0NjBlM2UxOGE2MzQxOWU1MTlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.25Vqhuq7GtimiCffl6A9lqdxH31KAYxw44Kdnz6d1UE)

Its possible your ISP is blocking it for some reason. Try a VPN. You could also just download straight from GitHub.

https://github.com/shdwmtr/steambrew/blob/main/docs/ui/users/installing.md

**Reply by DKTGM** [reply]
I actually have tried to install it, but I get a whole error in red text saying that the operation is invalid
![image](https://github.com/user-attachments/assets/f4a4dcd8-3327-4bf7-85fd-1554f1d0ffe7)

**Reply by DKTGM** [reply]
It also says that "iwr" is not an internal or external command for me on the CMD

**Reply by ejalxndr** [reply]
Oops, I missed responding. try:
`iwr -useb "https://raw.githubusercontent.com/shdwmtr/millennium/refs/heads/main/scripts/install.ps1" | iex`

**Reply by DKTGM** [reply]
I'll try that lol

**Reply by DKTGM** [reply]
It works! Thanks so much!

**Reply by ejalxndr** [reply]
@DKTGM Keep in mind since your ISP blocks Millenniums servers, you won't receive updates. 

You should take a look into why your ISP blocks steambrew.app
